src/main/java/sg/ncl/testbed_interface/DataAccessRequest.java

```diff
@@ -0,0 +1,43 @@
+package sg.ncl.testbed_interface;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Created by dcsjnh on 12/27/2016.
+ */
+@Getter
+@Setter
+public class DataAccessRequest implements Serializable {
+
+    private Long id;
+    private Long dataId;
+    private String requesterId;
+    private String reason;
+    private ZonedDateTime requestDate;
+    private ZonedDateTime approvedDate;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this value-based field transient so it is not included in the serialization of this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3437)